libs/blocks/global-navigation/global-navigation.js

```diff
@@ -561,7 +562,11 @@ class Gnav {
           <div class="feds-navItem${isSectionMenu ? ' feds-navItem--section' : ''}">
             ${dropdownTrigger}
           </div>`;
-        dropdownTrigger.addEventListener('click', (e) => trigger({ element: dropdownTrigger, event: e }));
+        dropdownTrigger.addEventListener('click', (e) => {
+          trigger({ element: dropdownTrigger, event: e });
+          const isOpen = dropdownTrigger.getAttribute('aria-expanded') === 'true';
+          if (isOpen) triggerTemplate.classList.add(selectors.activeDropdown.replace('.', ''));
```

Comment:
`trigger()` actually returns `true|false` based on if it was opened or not, so you don't need the isOpen check

```suggestion
          const hasBeenOpened = trigger({ element: dropdownTrigger, event: e });
          if (hasBeenOpened) triggerTemplate.classList.add(selectors.activeDropdown.replace('.', ''));
```